data-processing/entities/definitions/commands/detect_definitions.py

```diff
@@ -926,11 +923,14 @@ def process(
                 yield definiendum
 
         # Detect all other references to the defined terms. Detect references to textual
-        # terms, abbreviations, and symbols.
+        # terms and abbreviations. References to symbols need not be found here; they
+        # will be detected automatically in the symbol extraction code.
         term_index = 0
 
         for tex_path, file_contents in item.tex_by_file.items():
-            term_extractor = PhraseExtractor(term_phrases + abbreviations + symbol_nicks)
+            term_extractor = PhraseExtractor(
+                term_phrases + abbreviations
```

Comment:
@andrewhead Is there a reason we don't want symbol-nick definitions to get aggregated as well?

Reply:
@nuwandavek yeah and great question! (I explain in a bit more detail in the comment just above this block)---all symbol usages will get detected automatically in the symbol extraction part of the pipeline, making detection of references to symbols in this part of the code redundant.